src/back/glsl/features.rs

```diff
@@ -193,7 +193,7 @@ impl<'a, W> Writer<'a, W> {
             match ty.inner {
                 TypeInner::Scalar { kind, width } => self.scalar_required_features(kind, width),
                 TypeInner::Vector { kind, width, .. } => self.scalar_required_features(kind, width),
-                TypeInner::Matrix { .. } => self.scalar_required_features(ScalarKind::Float, 8),
+                TypeInner::Matrix { .. } => self.scalar_required_features(ScalarKind::Float, 4),
```

Comment:
This should be
```rust
TypeInner::Matrix {width, .. } => self.scalar_required_features(ScalarKind::Float, width),
```

Reply:
uuh, how did I miss that :) thanks for spotting!